django_mailbox/models.py

```diff
@@ -498,10 +498,11 @@ def reply(self, message):
         pre-set it.
 
         """
-        if self.mailbox.from_email:
-            message.from_email = self.mailbox.from_email
-        else:
-            message.from_email = settings.DEFAULT_FROM_EMAIL
+        if not getattr(message, 'from_email', None):
```

Comment:
I think ```message.from_email``` should be never empty, so I want to overwrite if empty too. This is a bit different than ```if not hasattr(message, 'from_email')```. However, how will you decide.